src/validation/document-definitions-validator.js

```diff
@@ -0,0 +1,53 @@
+var environmentShellMaker = require('../document-definitions-shell-maker');
```

Comment:
This is intended to take the place of the work-in-progress `src/document-definitions-validator` module (notice the difference in file paths). The WIP module will be removed in a different pull request.